Corporate Mining outpost

LordNest opened the pull request:
Another one mining map

![2023-09-21 17 22 36](https://github.com/Baystation12/Baystation12/assets/77477080/420a7dc6-ded2-4978-90bd-4b3588a1a353)


:cl: LordNest
maptweak: Added another away map with a crashed shuttle on it
/:cl:

LordNest committed:
Corporate Mining outpost

Another one mining map

SierraKomodo commented on maps/away/mining/mining.dm:
```diff
@@ -242,3 +242,150 @@
 	color = "#c9ae5e"
 	map_airless = TRUE
+
+//MINING-4 //CORPORATE
```
This file is starting to get a bit lengthy and bloated - Let's do a new file for the corporate map specific stuff.

SierraKomodo commented on maps/away/mining/mining_areas.dm:
```diff
@@ -24,3 +24,57 @@
 	icon_state = "LP"
 	turfs_airless = TRUE
+
+// CORPORATE OUTPOST
```
As above, let's start doing separate files for separate maps.

SierraKomodo commented on maps/away/mining/mining_areas.dm:
```diff
@@ -24,3 +24,57 @@
+/area/outpost/mining/solar
+	name = "Mining Outpost Solar Array"
+	area_flags = AREA_FLAG_EXTERNAL
+	requires_power = 1
+	always_unpowered = 1
+	has_gravity = FALSE
```
Use booleans for boolean values.

```suggestion
	requires_power = TRUE
	always_unpowered = TRUE
	has_gravity = FALSE
```

SierraKomodo reviewed (changes requested):
Code review. Map review will be done separately.

SierraKomodo reviewed (dismissed):
Map review:
- Is there a specific reason the fax machine is locked to mining access? No other fax machines have access locks in the code base currently.
- Coordinate 129,122 has a space tile in a mining area - This should either be a space area or an asteroid turf. ![StrongDMM_oumyRuqHjM](https://github.com/Baystation12/Baystation12/assets/11140088/696ad9dd-81b8-4ede-a825-3fdcbda11099) ![StrongDMM_lFO4xW6MOU](https://github.com/Baystation12/Baystation12/assets/11140088/320588c3-57ad-41e1-9200-815bae68fc14)
- Coordinate 121,126 is missing a power cable. ![StrongDMM_IPy1ksJx0n](https://github.com/Baystation12/Baystation12/assets/11140088/c1daf77a-c7e4-4695-9c84-ab1dfffcda82)

LordNest commented:
![image](https://github.com/Baystation12/Baystation12/assets/77477080/e9c0af49-5145-435d-a7ff-7d22a24e5437)
![image](https://github.com/Baystation12/Baystation12/assets/77477080/9c05bc1c-9b43-4b53-9440-af1864764d96)

Fixed map issues. Fixed bloated code issues
